app/uk/gov/hmrc/economiccrimelevyaccount/config/AppConfig.scala

```diff
@@ -37,6 +37,7 @@ class AppConfig @Inject() (configuration: Configuration, servicesConfig: Service
   val signInUrl: String       = configuration.get[String]("urls.signIn")
   val signOutUrl: String      = configuration.get[String]("urls.signOut")
   val registrationUrl: String = configuration.get[String]("urls.registration")
+  val returns: String         = configuration.get[String]("urls.returns")
```

Comment:
Sorry to be pedantic, but can we name this "returnsUrl" to be consistent with the others please :)